website/package.json

```diff
@@ -52,6 +52,7 @@
         "react-toastify": "^11.0.2",
         "react-tooltip": "^5.28.0",
         "rsuite": "^5.76.2",
+        "string-similarity-js": "^2.1.4",
```

Comment:
Given that the library doesn't seem to be actively maintained (last commit 4 years ago, and not many stars) and just consists of [one file](https://github.com/stephenjjbrown/string-similarity-js/blob/master/src/string-similarity.ts), I wonder whether it would be better to simply copy over the code (of course with a reference back to the repository) instead of including the package to reduce the surface for supply chain attacks.